web-server/v0.4/README.md

```diff
@@ -61,13 +61,7 @@ This will automatically open the application on [http://localhost:8000](http://l
 
 ## Local Development
 
-Both the production and development builds of the dashboard require specific configurations in order to run on their respective environment. For local development, modify the following configuration definitions:
-
-`.webpackrc.js`
-
-```Javascript
-publicPath: '/'
-```
+Both the production and development builds of the dashboard require specific configurations in order to run on their respective environment. For local development, modify the following configuration definition:
```

Comment:
Does the last sentence in this comment need to change now?

Reply:
We before you had lines 65-70 in place, but now you removed them leaving the last line reading, "For local development, modify the following configuration definition:", but nothing appears to be there.